### PR TITLE
Adds grunting

### PR DESCRIPTION
**Pull request by LordFowl**
Adds a new emote, *grunt, which makes the player character grunt, to human mobs, for the purpose of allowing players to allow their character to grunt.

### CODE REVIEW

**Comment by LordFowl** [comment]
!review

**Comment by dragoslava** [comment]
> bad sound BAD SOUND

I fail to see the issue

**Comment by ParadoxSpace** [comment]
It's a meme sound.

File: html/changelogs/aooeee.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Adds a new emote to human mobs."
```
Comment (Sindorman) [inline html/changelogs/aooeee.yml]:
```suggestion
  - rscadd: "Adds a new emote to human mobs."
  - soundadd: "Added grunting sound."
```

**Comment by VTCobaltblood** [comment]
You can already grunt by emoting. What's the purpose of this exactly? Should we also make screams produce sounds? Should we make speech produce sounds?

**Comment by Sindorman** [comment]
@VTCobaltblood I am not sure about this specific sound. But I would honestly love if we add screaming sounds for when people automatically scream in pain. I also want to add some other sounds, but it is not related to this PR.

**Comment by LordFowl** [comment]
This is what grunting sounds like?

**Comment by skull132** [comment]
Sound's a no-go.

**Pull request closed without merging**